frontend/src/js/components/devices/device-groups.js

```diff
@@ -147,7 +147,7 @@ export const DeviceGroups = () => {
     }
     // preset selectedIssues and selectedId with empty values, in case if remain properties are missing them
     let listState = { ...remainder };
-    if (statusParam && Object.values(DEVICE_STATES).some(state => state === statusParam)) {
+    if (statusParam && Object.values(DEVICE_STATES).some(state => state === statusParam || statusParam === 'any')) {
```

Comment:
```suggestion
    if (statusParam && (Object.values(DEVICE_STATES).some(state => state === statusParam) || statusParam === routes.allDevices.key)) {
```
to not have the check in every iteration there